Sources/Apodini/Properties/Binding.swift

```diff
@@ -45,6 +47,23 @@ public struct Binding<Value>: DynamicProperty, PotentiallyParameterIdentifyingBi
             return retriever(store)
         }
     }
+    
+    public var projectedValue: Self {
+        self
```

Comment:
Why do you want to expose the `Binding` itself via the `projectedValue`? We can already use the underscore `_someBinding` for quick access...why do we need `$someBinding` on top of that? This only results in a breaking change later on if we wanted to use the `projectedValue` for something else.

Reply:
The goal of the projected value was to provide the same "just add a $ and you have a binding" (https://developer.apple.com/documentation/swiftui/binding/projectedvalue) approach that SwiftUI is approaching for `@State` and `@Binding`. I was preparing a session to show Apodini to the iPraktikum session tomorrow and noted this divergence. At the moment I don't see any other usage for the projected value.
Using this enables you to write the projected value for the following two cases. I am convinced this makes it a bit easier to reason about for newcomers:
```swift
struct MyComponent: Component {
    @PathParameter var id: Int


    var content: some Component {
         Group($id) {
             MySecondComponent(id: $id)
         }
    }
}


struct MySecondComponent: Component {
    @Binding var id: Int


    var content: some Component {
         Group($id) {
             MyHandler(id: $id)
         }
    }
}

struct MyHandler: Handler {
    @Binding var id: Int


    func handle() -> Empty { /*...*/ }
}
```